components/edit-collective/Form.js

```diff
@@ -479,7 +479,10 @@ class EditCollectiveForm extends React.Component {
               <FormattedMessage id="Account.AdvancedSettings" defaultMessage="Advanced settings" />
             </SettingsTitle>
             {collective.type === CollectiveType.USER && <EditUserEmailForm />}
-            {(collective.type === CollectiveType.COLLECTIVE || collective.type === CollectiveType.FUND) && (
+            {(collective.type === CollectiveType.COLLECTIVE ||
```

Comment:
Code style, do something like `[COLLECTIVE, FUND, PROJECT, EVENT].includes(collective.type);`

Reply:
Done. 👍🏽

Reply:
Don't see the change.

Reply:
@znarf : Ops sorry, I've changed this on `EmptyBalance.js` but missed it on the `Form.js`. Just corrected it. 👍🏽